File: coding/python/fizzbuzz.py
class fizbuzz:

    def __init__(self, start, end):
        self.start = start
        self.end = end

    def print_fizzbuzz(self):

        for i in range(self.start, self.end + 1):

            if i % 3 == 0 and i % 5 == 0:
                print("FizzBuzz")

            elif i % 3 == 0:
                print("Fizz")

            elif i %5 == 0:
                print("Buzz")

            else:
                print(i)

fb = fizbuzz(1, 100)

fb.print_fizzbuzz()

File: coding/java/fizzbuzz.java
package coding.java;

public class fizzbuzz {
    
    public static void main(String[] args) {
        printFizzBuzz(100);
    }

    public static void printFizzBuzz(int N) {
        for (int i =1; i <=N; i++) {
            if ((i % 3 == 0) && (i % 5 == 0)) {
                System.out.println("FizzBuzz");
            }else if (i % 3 == 0){
                System.out.println("Fizz");
            }else if (i % 5 == 0){
                System.out.println("Buzz");
            }else{
                System.out.println(i);
            }
        }
    }
}
